src/content/public-goods-funding/index.md

```diff
@@ -0,0 +1,304 @@
+
+Flexibility also reduces the overhead of coordinating public goods funding on a global scale. For example, parameters of a payment splitter or streaming contract (e.g., amounts donated, allocation weights, allocation timeline, etc.), can be routinely adjusted either by donors, trusted entities, or a group (via [on-chain governance](/dao/#dao-governance)). This reduces bureaucracy, increases autonomy for donors, and improves the efficiency of initiatives aimed at financing public goods.
+
+### Incentive alignment {#incentive-alignment}
+
+Public goods markets on Ethereum are designing better incentives to encourage greater public investment in digital public goods. Part of this approach involves designing better systems for measuring public opinion on what public goods deserve funding.
+
+For example, some Ethereum-based public goods markets use a quadratic voting system to control the allocation of funds to organizations building digital public goods. [Quadratic voting](https://vitalik.ca/general/2019/12/07/quadratic.html) is considered ideal because it strikes a balance between one-person-one-vote and one-dollar-one-vote systems.
+
+Below is an overview of how the combination of quadratic voting with public fundraising (**quadratic funding**) works:
+
+1. Individuals and organizations donate funds to a "matching pool" at the beginning of a funding round. These funds are distributed to different public goods projects at the end of the round based on their weighted share of votes.
+
+2. Anyone can "vote" for a project by donating some amount to it during the funding round. The influence of each voter is calculated as the square root of the amount donated. If Alice donates $100, her influence on the outcome equals that of ten people who contribute $1.
+
+3. Every donation towards a project is matched with a corresponding amount of funding from the official matching pool. Instead of being matched 1:1, funds are matched according to the [quadratic funding formula](https://wtfisqf.com/?grant=10&grant=5,5&grant=2,2,2,2,2&grant=1,1,1,1,1,1,1,1,1,1&match=1000).
+
+The formula works by adding the square roots of each contribution, squaring the result, and subtracting the original amount donated from it. The final figure represents the matched funding, i.e., how much money each project receives from the matching pool. We can explain this concept with an example:
+
+Imagine two parks are planned for development, with the teams building each park (we call them `park A` and `park B`) requesting for funding. Ten people agree to donate $25 each to build `park A`, while two affluent investors donate $180 each to `park B`.
+
+The table below shows how much each park will receive under a quadratic funding scheme:
+
+![A diagram showing showing funds received during a quadratic funding round](./quadratic-funding-table.png)
```

Comment:
I'm not sure I see the benefit of a table as an image when we could render this as text and translate it.
I would suggest using a table here:
```suggestion
| | No. of donors | Amount contributed by individual donors | Total contribution | Matched funding | Total funding received |
| :--- | :--- | :--- | :--- | :--- | :--- |
| Park A | 10 | $25 | $250 | $2,250 | $2,500 |
| Park B | 2 | $400 | $800 | $800 | $1,600 |
```

Reply:
+1. Let me know if you want me to do this @emmanuel-awosika.

Reply:
Looks good to me.